packages/e2e/cypress/integration/ui/theme/theme-selector/theme-selector.steps.ts

```diff
@@ -0,0 +1,19 @@
+import { And } from 'cypress-cucumber-preprocessor/steps';
+import { escapeRegExp } from 'lodash';
+
+And('I change the theme to {string}', (name) => {
+  cy.findByRole('button', {
+    name: new RegExp(`${escapeRegExp(name)}`, 'i'),
+  })
+    .should('be.visible')
+    .click({ force: true });
```

Comment:
Cypress was having trouble clicking the theme buttons, because they were covered in `svgs`:
![Screen Shot 2022-03-16 at 1 31 42 PM](https://user-images.githubusercontent.com/43682783/158685732-dd8e3e53-2723-4524-971a-0e52407ad5e2.png)

Reply:
That `svg` is actually from the light/dark mode toggle button, not sure why cypress think it's covering the buttons. Maybe because it's on header?

Reply:
IME cypress tests with click force true tend to be flaky. Could we avoid the problem of the header getting in the way by jumping to a section a little higher on the page before trying to click the button?

Reply:
Ahhh, I get why the test is failing now :p Let me try. Is there a helper cypress function that lets me do that?

Reply:
Ended up using `.click({ scrollBehavior: 'center' })` so that cypress scrolls far enough to have these buttons below the header.

It works pretty well!

<img width="1904" alt="image" src="https://user-images.githubusercontent.com/43682783/158692419-f3e08b66-bfcc-4125-a6d8-824879bfeaed.png">

Reply:
Done in 2340122b7cc6e7d883c53ab78b30ee22d8bf0158